packages/loader/src/ktx2/KTX2Container.ts

```
@@ -52,8 +53,15 @@ export class KTX2Container {
-  get isUASTC() {
-    return this.dataFormatDescriptor.colorModel === ColorModel.UASTC;
+  get colorModel() {
+    switch (this.dataFormatDescriptor.colorModel) {
+      case ColorModel.ETC1S:
+        return "etc1s";
+      case ColorModel.UASTC:
+        return "uastc";
+      case ColorModel.HDR:
+        return "hdr";
+    }
   }
```

Biome (1.9.4)

[error] This getter should return a value.



(lint/suspicious/useGetterReturn)